src/Calculator/App.xaml.cs

```diff
@@ -423,9 +423,9 @@ private async Task SetupJumpList()
                         continue;
                     }
                     ViewMode mode = option.ViewMode;
-                    var item = JumpListItem.CreateWithArguments(((int)mode).ToString(), "ms-resource:///Resources/" + NavCategoryStates.GetNameResourceKey(mode));
+                    var item = JumpListItem.CreateWithArguments(mode.ToString(), "ms-resource:///Resources/" + NavCategoryStates.GetNameResourceKey(mode));
```

Comment:
Enum.ToString() has different behavior than int.ToString(). Is the new behavior correct?

Reply:
By bad. I fixed this.